config/nautobot_config.py

```diff
@@ -0,0 +1,134 @@
+"""Nautobot development configuration file."""
+# pylint: disable=invalid-envvar-default
+import os
+import sys
+
+from nautobot.core.settings import *  # noqa: F403  # pylint: disable=wildcard-import,unused-wildcard-import
+from nautobot.core.settings_funcs import parse_redis_connection, is_truthy
+
+
+#
+# Misc. settings
+#
+
+ALLOWED_HOSTS = os.getenv("NAUTOBOT_ALLOWED_HOSTS", "").split(" ")
+SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "")
```

Comment:
These are defaults in `nautobot.core.settings` and shouldn't need to be explicitly stated here.

Reply:
Done.